README.md

```diff
@@ -1866,7 +1866,7 @@ void AGDCharacterBase::AddCharacterAbilities()
 			FGameplayAbilitySpec(StartupAbility, GetAbilityLevel(StartupAbility.GetDefaultObject()->AbilityID), static_cast<int32>(StartupAbility.GetDefaultObject()->AbilityInputID), this));
 	}
 
-	AbilitySystemComponent->CharacterAbilitiesGiven = true;
+	AbilitySystemComponent->bCharacterAbilitiesGiven = true;
```

Comment:
If we change it here, we should change it in the sample project code too.

Reply:
Unless I'm missing something, it's written that way in the sample project code (checked master and 5.1 branches).

Reply:
https://github.com/tranek/GASDocumentation/blob/master/Source/GASDocumentation/Private/Characters/GDCharacterBase.cpp#L288